runtime/development/src/lib.rs

```diff
@@ -836,7 +836,7 @@ impl pallet_pools::Config for Runtime {
 	type TrancheId = TrancheId;
 	type EpochId = u32;
 	type CurrencyId = CurrencyId;
-	type Tokens = OrmlTokens;
+	type Tokens = Tokens;
```

Comment:
This should be tokens. Otherwise, one can not use a native currency in the pools.